### PR TITLE
[chrome-api-jstocxxtest] Test EndTransaction

### PR DESCRIPTION
**Pull request by ViktoriiaKovalova**

### CODE REVIEW

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 1]
```diff
@@ -867,6 +867,34 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+
+      expectReportPlainResult(/*requestId=*/ 112, 'SUCCESS');
+      await mockChromeApi
+          .dispatchEvent(
+              'onEndTransactionRequested', /*requestId=*/ 112, readerHandle,
+              chrome.smartCardProviderPrivate.Disposition.LEAVE_CARD)
+          .$waitAndVerify();
+    },
+
+
```
Comment (emaxx-google) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:888]: nit: Remove the extra blank line.

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js [hunk 2]
```diff
@@ -867,6 +867,34 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
...
+    // Test that EndTransaction fails without BeginTransaction.
+    'testEndTransaction_fail': async function() {
+      expectReportPlainResult(/*requestId=*/ 111, 'NOT_TRANSACTED');
```
Comment (emaxx-google) [inline smart_card_connector_app/src/chrome-api-jstocxxtest.js:891]: As discussed offline, it's a bit of a weird error code, but maybe that's what PC/SC-Lite always been doing. I'll add a similar C++ test (seems good anyway) and try to track down the code.

**Review by emaxx-google — APPROVED** [review]

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 1 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js]: thanks, fixed

File: smart_card_connector_app/src/chrome-api-jstocxxtest.js (hunk 2 above)
Comment (ViktoriiaKovalova) [reply smart_card_connector_app/src/chrome-api-jstocxxtest.js:890]: thanks, let's do this :)

**Pull request merged**